spec/unit/prepare.spec.js

```diff
@@ -21,6 +21,8 @@ const rewire = require('rewire');
 const path = require('path');
 const CordovaError = require('cordova-common').CordovaError;
 const GradlePropertiesParser = require('../../lib/config/GradlePropertiesParser');
+const utils = require('../../lib/utils');
+const et = require('elementtree');
```

Comment:
`elementtree` is not actually declared by `cordova-android` as a dependency.. The dependency is available through cordova-common but if cordova-android needs to use it, it should have it's own npm install to avoid the hidden dependency.

Can you `npm install elementtree` it for this PR?

Reply:
Make sure you install it as a devDependency.

`npm install -D elementtree`

From this PR, it is only used in testing.

Reply:
Good catch..

Reply:
elementtree dependency added

Reply:
Thanks!